src/AnalyzerConfiguration/NI.CSharp.Analyzers.targets

```diff
@@ -39,16 +39,17 @@
     <Analyzer Include="$(PkgStyleCop_Analyzers)\analyzers\dotnet\cs\StyleCop.Analyzers.dll"/>
     <Analyzer Include="$(PkgStyleCop_Analyzers)\analyzers\dotnet\cs\StyleCop.Analyzers.CodeFixes.dll"/>
 
-    <RoslynAnalyzer Include="Microsoft.CodeQuality" />
-    <RoslynAnalyzer Include="Microsoft.NetCore" />
-    <RoslynAnalyzer Include="Microsoft.NetFramework" />
-    <RoslynAnalyzer Include="Text" />
-
     <!-- Microsoft's CA analyzers -->
-    <Analyzer Include="@(RoslynAnalyzer->'$(NuGetPackageRoot)\%(Identity).analyzers\2.6.3\analyzers\dotnet\cs\%(Identity).Analyzers.dll')"/>
-    <Analyzer Include="@(RoslynAnalyzer->'$(NuGetPackageRoot)\%(Identity).analyzers\2.6.3\analyzers\dotnet\cs\%(Identity).CSharp.Analyzers.dll')"/>
-    <Analyzer Include="$(PkgMicrosoft_VisualStudio_Threading_Analyzers)\analyzers\cs\Microsoft.VisualStudio.Threading.Analyzers.dll"/>
     <Analyzer Include="$(PkgMicrosoft_CodeQuality_Analyzers)\analyzers\dotnet\cs\Humanizer.dll" />
```

Comment:
Where are these properties (PkgMicrosoft_*) defined?

Reply:
I'm assuming there is a <PackageReference> to these particular packages that are sucked in via dependencies

Reply:
Correct. They are created by the system.

Reply:
Weird. I see them in <Project>.csproj.nuget.g.props; not sure how those are created. How did you find them?

Reply:
There were already several being used in other places in this file. This is also the standard form that is created when you use GeneratePathProperty with PackageReferences. Here they seem to be available even without that mechanism. There is a [comment in this file](https://github.com/ni/csharp-styleguide/blob/main/build/PathPropertyGeneration.txt) that says the variable is automatically created for any package containing a Tools directory.